src/main/java/org/example/racinggame/controller/RacingCarController.java

```diff
@@ -1,38 +1,40 @@
 package org.example.racinggame.controller;
 
+import java.util.List;
+
 import org.example.racinggame.model.Cars;
 import org.example.racinggame.view.InputView;
 import org.example.racinggame.view.OutputView;
 
 public class RacingCarController {
-
-    private final InputView inputView;
-    private final OutputView outputView;
-
-    public RacingCarController(InputView inputView, OutputView outputView) {
-        this.inputView = inputView;
-        this.outputView = outputView;
+    private final Cars cars;
+    private final String carNames;
+    private final int tryCount;
+    private int currentCount = 0;
+
+    public RacingCarController(String carNames, int tryCount) {
+        this.carNames = carNames;
+        this.tryCount = tryCount;
+        cars = new Cars(carNames);
     }
 
     public void play() {
-        String carNames = inputView.getCarNamesFromUserInput();
-        int tryCount = inputView.getTryCountFromUserInput();
-
-        Cars cars = new Cars(carNames);
+        cars.move();
+        currentCount++;
```

Comment:
tryCount를 역으로 하나씩 감소시킨다면  `currentCount`를 인스턴스 변수로 둘 필요가 없을 것 같네요. 
가능하다면 객체가 최소한의 인스턴스 변수만 가지도록 해주세요. 🙏

```suggestion
        tryCount--;
```